WooCommerce/Classes/ViewRelated/MainTabBarController.swift

```diff
@@ -477,26 +461,16 @@ private extension MainTabBarController {
         let productsViewController = createProductsViewController(siteID: siteID)
         productsNavigationController.viewControllers = [productsViewController]
 
-        // Configure hub menu tab coordinator or reviews tab coordinator once per logged in session potentially with multiple sites.
-        if isHubMenuFeatureFlagOn {
-            if hubMenuTabCoordinator == nil {
-                let hubTabCoordinator = createHubMenuTabCoordinator()
-                self.hubMenuTabCoordinator = hubTabCoordinator
-                hubTabCoordinator.start()
-            }
-            hubMenuTabCoordinator?.activate(siteID: siteID)
-        }
-        else {
-            if reviewsTabCoordinator == nil {
```

Comment:
nit: maybe we can also remove this variable, and the whole `ReviewsCoordinator` if it's not used anymore?

Reply:
Looks like we're not using it anymore indeed, I'll work on this separately here: https://github.com/woocommerce/woocommerce-ios/issues/7643